src/components/admin/AdminView.tsx

```diff
@@ -106,7 +119,7 @@ const AdminView = () => {
         </Flex>
       </Flex>
       {locations.map(location => (
-        <AdminCard key={location.id} assignmentOrLocation={location} editMutation={editLocationMutation} />
+        <AdminCard key={location.id} assignmentOrLocation={location} editMutation={editLocationMutation} handleDelete={handleDeleteLocation} />
       ))}

```

Comment:
Pass in the approprite handler as a prop. The top is for assignments and the bottom is for locations.